orbit/cmd/desktop/desktop.go

```diff
@@ -404,6 +372,23 @@ func main() {
 					if err := open.Browser(openURL); err != nil {
 						log.Error().Err(err).Str("url", openURL).Msg("open browser my device")
 					}
+					// Also refresh the device status
+					sum, err := client.DesktopSummary(tokenReader.GetCached())
```

Comment:
May this race with the other routine also getting the desktop summary and updating the buttons? 

An alternative is to signal the goroutine that does the 5m thing to do the check "now" via some channel.

Reply:
Yes, I like the alternative and will try to implement that.

Reply:
@lucasmrod I made the improvement. Please re-approve